.github/workflows/zowe-cli.yml

```diff
@@ -26,8 +26,11 @@ on:
 
 jobs:
   test:
-    name: Cross-Platform Test
+    if: (github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository) && !contains(github.event.head_commit.message, '[ci skip]')
```

Comment:
Is there a difference in where the `if` is placed ?
Just curious.
I kind of like it being at the very top, makes "mental debugging" easier 😋

Reply:
I don't think it makes a difference, more just personal preference and consistency with other repos 😛